modules/apps/depot/depot-test/src/testIntegration/java/com/liferay/depot/web/internal/asset/model/test/DepotAssetRendererFactoryTest.java

```diff
@@ -130,17 +132,14 @@ null, null, new UnsyncByteArrayInputStream(bytes), bytes.length,
 	public void testGetAssetRendererJournalArticleWithInvalidGroup()
 		throws Exception {
 
-		JournalArticle journalArticle = JournalTestUtil.addArticle(
-			_depotEntry.getGroupId(),
-			JournalFolderConstants.DEFAULT_PARENT_FOLDER_ID,
-			Collections.emptyMap());
-
-		ServiceContext serviceContext = new ServiceContext();
-
-		serviceContext.setScopeGroupId(RandomTestUtil.randomLong());
+		ServiceContext serviceContext = _getServiceContext();
 
 		ServiceContextThreadLocal.pushServiceContext(serviceContext);
 
+		JournalArticle journalArticle = JournalTestUtil.addArticle(
+			_depotEntry.getGroupId(),
+			JournalFolderConstants.DEFAULT_PARENT_FOLDER_ID, serviceContext);
```

Comment:
Since this test should work with an invalid groupId, you may not need to change to call the method that passes the serviceContext.